ee/modules/030-cloud-provider-openstack/templates/csi/controller.yaml

```diff
@@ -69,6 +69,7 @@
   {{- $csiNodeConfig := dict }}
   {{- $_ := set $csiNodeConfig "nodeImage" $csiControllerImage }}
   {{- $_ := set $csiNodeConfig "driverFQDN" "cinder.csi.openstack.org" }}
+  {{- $_ := set $csiNodeConfig "serviceAccount" "csi" }}
```

Comment:
Maybe we should change default value in `helm_lib_csi_node_manifests` template?

Reply:
Good point. I'll fix the template.